.travis.yml

```diff
@@ -1,40 +0,0 @@
-sudo: required
-services:
-  - docker
-
-before_install:
-  - docker pull gcr.io/cpp-projects/cpp-ci:1
-
-script:
-  - >
-    docker run -v `pwd`:/leatherman gcr.io/cpp-projects/cpp-ci:1 /bin/bash -c "
-    cd /leatherman &&
-    rm locales/leatherman.pot &&
-    cmake $EXTRA_VARS . &&
-    mkdir dest &&
-    make $TARGET DESTDIR=/leatherman/dest -j2 &&
-    { [ '$COVERALLS' == 'ON' ] && coveralls --gcov-options '\-lp' -r . -b . -e src -e vendor >/dev/null || true; }
-    "
-  - if [ "$DO_RELEASE" == "true" ]; then tar czvf leatherman${PKG_SUFFIX}.tar.gz `find dest -type f -print`; fi
-
-env:
-  matrix:
-    - TARGET=cpplint
-    - TARGET=cppcheck
-    - TARGET="all test install ARGS=-v" DO_RELEASE=true PKG_SUFFIX="" EXTRA_VARS="-DBOOST_STATIC=ON"
-    - TARGET="all test install ARGS=-v" DO_RELEASE=true PKG_SUFFIX="-dynamic" EXTRA_VARS="-DLEATHERMAN_SHARED=ON"
-    - TARGET="all test install ARGS=-v" EXTRA_VARS="-DCMAKE_BUILD_TYPE=Debug -DCOVERALLS=ON" COVERALLS=ON
-    - TARGET="all test install ARGS=-v" EXTRA_VARS="-DLEATHERMAN_USE_LOCALES=OFF"
-    - TARGET="all test install ARGS=-v" EXTRA_VARS="-DLEATHERMAN_GETTEXT=OFF"
-
-deploy:
```

Comment:
I think this 'deploy' section is missing from your github action; I ended up creating an action to make the `.github/` code DRYer in this [PR](https://github.com/puppetlabs/cpp-hocon/pull/136). I think we probably want to follow this pattern here as well?